test/cpp/end2end/grpc_authz_end2end_test.cc

```diff
@@ -54,6 +69,40 @@ std::string ReadFile(const char* file_path) {
   return file_contents;
 }
 
+class TestAuditLogger : public AuditLogger {
+ public:
+  explicit TestAuditLogger(int* log_count) : log_count_(log_count) {}
+
+  absl::string_view name() const override { return kLoggerName; }
+  void Log(const AuditContext& context) override { *log_count_ += 1; }
+
+ private:
+  int* log_count_;
+};
+
+class TestAuditLoggerFactory : public AuditLoggerFactory {
+ public:
+  class Config : public AuditLoggerFactory::Config {
+    absl::string_view name() const override { return kLoggerName; }
+    std::string ToString() const override { return ""; }
```

Comment:
I'd be a fan of at least returning a sensible name here in case this gets put into some kind of format string

Reply:
My understanding is this is supposed to return the config. I changed it to return `{}`. I am thinking about making it crash if called.

Reply:
I like either of those